### DISCUSSION TITLE
About the format of sound record

### DISCUSSION THREAD

**Discussion by cross-hello** [gsantner/markor#1445] [Q&A] [answered]
The format of sound record default is *wmv*, it may be better if could be save as **m4a** or any other more effectives.

**Comment by gsantner** [comment, marked as answer]
Yes, Markor uses a fairly simple format - which is available from Android platform regardless of the Android version.  
For a more advanced format, we would need either to unsupport certain codecs on older devices (what I dislike), or add some kind of multimedia framework like ffmpeg (what I dislike even more due bloat).

Markor's main usage though is text editing, at least I have the feeling not much people even use the audio recording feature. Otherwise there would have been bug reports and feature requests already (this is the first one since introduction).

TL;DR is: Not planning a change here, and for interesting people - you can always convert it later on if you need it in some different format.

**Reply by cross-hello** [reply]
Ok, Thanks for your reply. 
I may convert it after when files transferred to laptop. 
And your work for the editor is good.  Thank you.
